src/main/java/dev/meyi/bn/commands/BazaarNotifierCommand.java

```diff
@@ -1,15 +1,15 @@
 package dev.meyi.bn.commands;
 
 import dev.meyi.bn.BazaarNotifier;
-import dev.meyi.bn.utilities.EnchantedCraftingHandler;
-import dev.meyi.bn.utilities.Defaults;
-import dev.meyi.bn.utilities.Suggester;
-import dev.meyi.bn.utilities.Utils;
+import dev.meyi.bn.utilities.*;
+

```

Comment:
Star imports are frowned upon as it hides what all is being used. Just import it normally please